system_metrics_collector/test/system_metrics_collector/test_linux_cpu_measurement.cpp

```diff
@@ -309,8 +322,8 @@ TEST_F(LinuxCpuMeasurementTestFixture, TestPublishMetricsMessage)
   // 250 ms: kProcSamples[9] is collected. last GetStatisticsResults() is of kProcSamples[8-9]
   data = test_measure_linux_cpu_->GetStatisticsResults();
   expected_cpu_active =
-    ComputeCpuActivePercentage(ProcessStatCpuLine(
-        kProcSamples[8]), ProcessStatCpuLine(kProcSamples[9]));
+    ComputeCpuActivePercentage(ProcessStatCpuLine(kProcSamples[8]),
+      ProcessStatCpuLine(kProcSamples[9]));
```

Comment:
This is now the only section where `ProcessStatCpuLine` is not on a new line

Reply:
Thanks, will address.